.github/workflows/bench-pr.yml

```diff
@@ -2,15 +2,16 @@ name: PR Benchmarks
 
 on:
   pull_request:
-    types: [ labeled, synchronize ]
```

Comment:
the diffing on this line with spaces on and off is hilarious. One person adds spaces the next one removes it